### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by DaCoolX**
#232

Adds a good chunk of missing translations
Also splits every inlined `content:` line into an appropriate multi-line counterpart

### CODE REVIEW

**Comment by MacSplody** [comment]
> Also splits every inlined `content:` line into an appropriate multi-line counterpart

It would probably be best to put those in a separate pull request.
A lot of those messages could be handled in better ways, so they may end up getting replaced.
But I don't have any issue with updating the format.

**Comment by sibir-ine** [comment]
@DaCoolX I also noticed that quite of a few of the translations were accidentally given the `en` code. ~~I'll go through & find every instance, but I wanted to let you know beforehand.~~ Actually I'll do that after you split the PR.

**Comment by DaCoolX** [comment]
> It would probably be best to put those in a separate pull request.

I think that this is not possible nor required anymore.

> @DaCoolX I also noticed that quite of a few of the translations were accidentally given the `en` code.

Thanks for the hint, those should be fixed now but of-course feel free to take another look.
Used my beginner regex skills to check for those and didn't find any remaining ones but I wouldn't solely rely on that myself.

File: masterlist.yaml [hunk 1]
```diff
@@ -11710,7 +12424,11 @@ plugins:
     inc: [ 'xulColovianHighlands_EV.esp' ]
     msg:
       - type: say
-        content: 'Use instead ''Trails of Cyrodiil Merged UL Colovian Highlands.esp''.'
+        content:
+          - lang: en
+            text: 'Instead use ''Trails of Cyrodiil Merged UL Colovian Highlands.esp''.'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 2]
```diff
@@ -16169,11 +17393,19 @@ plugins:
   - name: 'Leyawin Bridge.esp'
     msg:
       - type: say
-        content: 'Part of the Rowboats mod.'
+        content:
+          - lang: en
+            text: 'Part of the Rowboats mod.'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 3]
```diff
@@ -16188,7 +17420,11 @@ plugins:
   - name: 'Sexier Sheogorath DV.esp'
     msg:
       - type: say
-        content: 'German version.'
+        content:
+          - lang: en
+            text: 'German version.'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 4]
```diff
@@ -16263,33 +17511,61 @@ plugins:
   - name: 'FUQ Land.esp'
     msg:
       - type: say
-        content: 'Old, update to LoversCharactors.esp'
+        content:
+          - lang: en
+            text: 'Old, update to LoversCharactors.esp'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 5]
```diff
@@ -16347,7 +17639,11 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/48920' ]
     msg:
       - type: say
-        content: 'Don''t need to use this if you use MMM as it already includes dogs.'
+        content:
+          - lang: en
+            text: 'Don''t need to use this if you use MMM as it already includes dogs.'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 6]
```diff
@@ -16356,7 +17652,11 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/24614' ]
     msg:
       - type: say
-        content: 'Probably not compatible with OOO. Update to version 2.0+.'
+        content:
+          - lang: en
+            text: 'Probably not compatible with OOO. Update to version 2.0+.'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 7]
```diff
@@ -16507,19 +17823,31 @@ plugins:
   - name: 'Erics Crafting Mod.esp'
     msg:
       - type: say
-        content: 'If it doesn''t work move past your bashed patch (author recommends that placement but the tags added here should fix that... to be investigated)'
+        content:
+          - lang: en
+            text: 'If it doesn''t work move past your bashed patch (author recommends that placement but the tags added here should fix that... to be investigated)'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 8]
```diff
@@ -16507,19 +17823,31 @@ plugins:
...
       - Invent
       - Scripts
   - name: 'Merchant_Tim.esp'
     msg:
       - type: say
-        content: 'Needs Shivering Isles. Otherwise use ''Tim_No_SI''.'
+        content:
+          - lang: en
+            text: 'Needs Shivering Isles. Otherwise use ''Tim_No_SI''.'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 9]
```diff
@@ -16550,7 +17878,11 @@ plugins:
     tag: [ Scripts ]
     msg:
       - type: warn
-        content: 'Contains corrupt identical-to-master records. Automated cleaning wth TES4Edit will remove them.'
+        content:
+          - lang: en
+            text: 'Contains corrupt identical-to-master records. Automated cleaning with TES4Edit will remove them.'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 10]
```diff
@@ -17638,7 +19138,11 @@ plugins:
   - name: 'Comprehensive Style Collection RM.esp'
     msg:
       - type: say
-        content: 'Outdated. Update to latest version.'
+        content:
+          - lang: en
+            text: 'Outdated. Update to latest version.'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 11]
```diff
@@ -18480,7 +20000,11 @@ plugins:
   - name: 'Luchaire''s Dryad Race.esp'
     msg:
       - type: say
-        content: 'Use ''for MMM'' version if using MMM/FCOM.'
+        content:
+          - lang: en
+            text: 'Use ''for MMM'' version if using MMM/FCOM.'
+          - lang: en
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
          - lang: de
```

File: masterlist.yaml [hunk 12]
```diff
@@ -19570,8 +21142,11 @@ plugins:
     inc: [ 'Duke Patricks - Basic Combat Geometry.esp' ]
     msg:
       - type: say
-        content: 'Requires: Underdark: The Realm of Northdark.'
-  - name: 'GG_Valeria_Revived.esp'
+        content:
+          - lang: en
+            text: 'Requires: Underdark: The Realm of Northdark.'
+          - lang: de
+            text: 'Benötigt: Underdark: The Realm of Northdark.'
```
Comment (sibir-ine) [inline masterlist.yaml:21189]:
```suggestion
            text: 'Benötigt: Underdark: The Realm of Northdark.'
  - name: 'GG_Valeria_Revived.esp'
```
Deleted this line by accident.

**Comment by DaCoolX** [comment]
Should have batched those, oh well.
I assume we merge on commit anyway.

**Comment by sibir-ine** [comment]
> Should have batched those, oh well.
> I assume we merge on commit anyway.

Yeah, it'll get squashed down. #game-masterlists in the Discord is having a field day, though 😛

File: masterlist.yaml [hunk 13]
```diff
@@ -5547,7 +5642,11 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/10980' ]
     msg:
       - type: say
-        content: 'May cause unfixable animation bugs if used with ''Personality Idles'' or similar.'
+        content:
+          - lang: en
+            text: 'May cause unfixable animation bugs if used with ''Personality Idles'' or similar.'
+          - lang: de
+            text: 'Kann unbehebare Animations-Fehler verursachen wenn es zusammen mit ''Personality Idles'' oder vergleichbaren verwendet wird.'
```
Comment (pStyl3) [inline masterlist.yaml]: Please change `unbehebare` to `unbehebbare`.

File: masterlist.yaml [hunk 14]
```diff
@@ -6403,18 +6562,30 @@ plugins:
...
   - name: 'Vows and Covenants.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/20736' ]
     msg:
       - type: say
-        content: 'Conflicts with Deadly Reflex. Minor Problems.'
+        content:
+          - lang: en
+            text: 'Conflicts with Deadly Reflex. Minor Problems.'
+          - lang: de
+            text: 'Steht im Konflikt mit Deadly Reflex. Geringe Probleme '
```
Comment (pStyl3) [inline masterlist.yaml]: Please put a `.` at the end of the sentence.

File: masterlist.yaml [hunk 15]
```diff
@@ -6475,9 +6654,17 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/11278' ]
     msg:
       - type: say
-        content: 'May cause crashes in the IC Market district. May have been fixed. Needs verification.'
+        content:
+          - lang: en
+            text: 'May cause crashes in the IC Market district. May have been fixed. Needs verification.'
+          - lang: de
+            text: 'Kann Abstürze im IC Markt Abteil verursachen. Kann schon behoben sein. Brauch Überprüfung.'
```
Comment (pStyl3) [inline masterlist.yaml]: Please change `Brauch` to `Braucht`, or alternatively `Benötigt`.

File: masterlist.yaml [hunk 16]
```diff
@@ -6486,7 +6673,11 @@ plugins:
       - 'https://www.nexusmods.com/oblivion/mods/11351'
     msg:
       - type: say
-        content: 'May cause crashes in the IC Market district. May have been fixed. Needs verification.'
+        content:
+          - lang: en
+            text: 'May cause crashes in the IC Market district. May have been fixed. Needs verification.'
+          - lang: de
+            text: 'Kann Abstürze im IC Markt Abteil verursachen. Kann schon behoben sein. Brauch Überprüfung.'
```
Comment (pStyl3) [inline masterlist.yaml]: Please change `Brauch` to `Braucht`, or alternatively `Benötigt`.

File: masterlist.yaml [hunk 17]
```diff
@@ -8419,7 +8730,11 @@ plugins:
     inc: [ 'FCOM_Convergence.esm' ]
     msg:
       - type: say
-        content: 'Remember to use the SI or no-SI versions as appropriate (same file names).'
+        content:
+          - lang: en
+            text: 'Remember to use the SI or no-SI versions as appropriate (same file names).'
+          - lang: de
+            text: 'Denken Sie dran die SI oder nicht-SI Versionen zu nutzen wie angebracht (selbe Dateinamen).'
```
Comment (pStyl3) [inline masterlist.yaml]: Please change `dran` to `daran`.

File: masterlist.yaml [hunk 18]
```diff
@@ -8429,7 +8744,11 @@ plugins:
       - 'Francesco''s Optional New Items Add-On.esm'
     msg:
       - type: say
-        content: 'Remember to use the SI or no-SI versions as appropriate (same file names).'
+        content:
+          - lang: en
+            text: 'Remember to use the SI or no-SI versions as appropriate (same file names).'
+          - lang: de
+            text: 'Denken Sie dran die SI oder nicht-SI Versionen zu nutzen wie angebracht (selbe Dateinamen).'
```
Comment (pStyl3) [inline masterlist.yaml]: Please change `dran` to `daran`.

File: masterlist.yaml [hunk 19]
```diff
@@ -10260,24 +10815,44 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/1960' ]
     msg:
       - type: warn
-        content: 'Do not use. This mod is confirmed to break a number of vanilla quests. Delete immediately.'
+        content:
+          - lang: en
+            text: 'Do not use. This mod is confirmed to break a number of vanilla quests. Delete immediately.'
+          - lang: de
+            text: 'Nicht nutzen. Diese Mod bricht nachgewiesenermaßen eine gewisse Anzahl von Original-Quests zu brechen. Unverzüglich entfernen.'
```
Comment (pStyl3) [inline masterlist.yaml]: Please delete `zu brechen` at the end, it's redundant.

File: masterlist.yaml [hunk 20]
```diff
@@ -10260,24 +10815,44 @@ plugins:
...
   - name: 'Forgrum''s Travelogue v2.0.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/3842' ]
     msg:
       - type: warn
-        content: 'Do not use. This mod is confirmed to break a number of vanilla quests. Delete immediately.'
+        content:
+          - lang: en
+            text: 'Do not use. This mod is confirmed to break a number of vanilla quests. Delete immediately.'
+          - lang: de
+            text: 'Nicht nutzen. Diese Mod bricht nachgewiesenermaßen eine gewisse Anzahl von Original-Quests zu brechen. Unverzüglich entfernen.'
```
Comment (pStyl3) [inline masterlist.yaml]: Please delete `zu brechen` at the end, it's redundant.

File: masterlist.yaml [hunk 21]
```diff
@@ -10260,24 +10815,44 @@ plugins:
...
+            text: 'French, no English translation appears to be available.'
+          - lang: de
+            text: 'Französisch, keine Englische Übersetzung scheint verfügbar zu sein.'
       - type: warn
-        content: 'Do not use. This mod is confirmed to break a number of vanilla quests. Delete immediately.'
+        content:
+          - lang: en
+            text: 'Do not use. This mod is confirmed to break a number of vanilla quests. Delete immediately.'
+          - lang: de
+            text: 'Nicht nutzen. Diese Mod bricht nachgewiesenermaßen eine gewisse Anzahl von Original-Quests zu brechen. Unverzüglich entfernen.'
```
Comment (pStyl3) [inline masterlist.yaml:10849]: Please delete `zu brechen` at the end, it's redundant.

File: masterlist.yaml [hunk 22]
```diff
@@ -10551,7 +11130,11 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/27973' ]
     msg:
       - type: say
-        content: 'If not using WB you may have to move this above some item mods (thereby losing some changes from this) or lose some items from those mods.'
+        content:
+          - lang: en
+            text: 'If not using WB you may have to move this above some item mods (thereby losing some changes from this) or lose some items from those mods.'
+          - lang: de
+            text: 'Wenn WB nicht genutzt, Muss dieses womöglich vor anderen Gegenstände-Mods platziert werden (dadurch ein paar Änderungen davon verlierend) oder einige Gegenstände von diesen Mods verlieren.'
```
Comment (pStyl3) [inline masterlist.yaml]: Please change `Muss` to `muss`.

File: masterlist.yaml [hunk 23]
```diff
@@ -12193,7 +12968,11 @@ plugins:
     msg:
       - *requiresBashedPatch
       - type: say
-        content: 'Recommended to upgrade to Quiet Feet MAX 2.0, which does not require a plugin and is more compatible.'
+        content:
+          - lang: en
+            text: 'Recommended to upgrade to Quiet Feet MAX 2.0, which does not require a plugin and is more compatible.'
+          - lang: de
+            text: 'Empfohlen zu Quiet Feet MAX 2.0 upzugraden, welches kein Plugin voraussetzt and ist besser kompatibel.'
```
Comment (pStyl3) [inline masterlist.yaml]: Please change `and` to `und`.

File: masterlist.yaml [hunk 24]
```diff
@@ -15269,7 +16304,11 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/9758' ]
     msg:
       - type: say
-        content: 'If using HC decide which you want to have priority and move that one after the other (needs compatibility patch that is not currently available)'
+        content:
+          - lang: en
+            text: 'If using HC decide which you want to have priority and move that one after the other (needs compatibility patch that is not currently available)'
+          - lang: de
+            text: 'Wenn HC genutzt wird, entscheiden Sie welcher für Sie Priorität hat und verschieben Sie diesen vor dem anderen (brauch Kompatibilitäts-Patch der momentan nicht verfügbar ist)'
```
Comment (pStyl3) [inline masterlist.yaml]: Please change `brauch` to `braucht`, or alternatively `benötigt`.

**Comment by sibir-ine** [comment]
I think we should hold off on merging this until after we significantly refactor these messages.

**Comment by DaCoolX** [comment]
Fixes applied, thanks @pStyl3 

> I think we should hold off on merging this until after we significantly refactor these messages.

Wouldn't that mean the refactor would have to be applied twice, both on the masterlimit and this PR?
As opposed to merging and then refactoring it.

**Comment by sibir-ine** [comment]
> Wouldn't that mean the refactor would have to be applied twice, both on the masterlimit and this PR?

No, you'd just need to rebase. I suppose we can merge it first since it'll take a while to refactor all those messages. However, you do need to rebase now because of d384e5859af33b7d56413df7d78e0a9d510e009d.

**Comment by DaCoolX** [comment]
Please, feel free to check the rebase. I dunno if I did that right.

**Comment by sibir-ine** [comment]
> 
> 
> Please, feel free to check the rebase. I dunno if I did that right.

Yep, perfect 🙂

**Comment by sibir-ine** [comment]
Thanks!

**Pull request merged**